README.md

```diff
@@ -1 +1,18 @@
-# ODP
\ No newline at end of file
+# E2E-Test Framework
+
+##1. How to run the e2e test against :
+
+### a) Local CDAP sandbox:
+
+Provide the local CDAP sandbox URL in [connectionParameters.properties](src/main/resources/connectionParameters.properties) file.
+>cdfurl=http://localhost:11011/pipelines/ns/default/studio
+
+Other keys (like @TC-Demo-1_GCS, Data-Set, Project-ID etc.) can be modified as needed. <br>
```

Comment:
Looks like [@TC-Demo-1_GCS](https://github.com/cdapio/cdap-e2e-tests/blob/develop/src/main/resources/connectionParameters.properties#L7) is going to be removed - remove reference to it. Also, please add comments in connectionParameters.properties indicating what each property is used for.

Reply:
Removed. And we will take this in a separate PR and add we have taken this in our backlog.